modules/core/package.json

```diff
@@ -44,6 +44,7 @@
     "@types/mocha": "^5.2.6",
     "@types/nock": "^9.3.1",
     "@types/node": "^11.11.4",
+    "algosdk": "git+https://github.com/BitGo/algosdk-bitgo.git#0668c1412466beff2591a94c10c8bad06e21d872",
```

Comment:
I don't think you need to reference this particular commit, it's the latest one.

Reply:
hmm I don't know if this is matters, I can change it, it pins to what's in teh the yarn lock i'm pretty sure

Reply:
OK, if that's the case then lgtm

Reply:
Don't think we need to pin a hash here - that's what the yarn.lock is for